gettsim/parameters/arbeitsl_geld_2.yaml

```diff
@@ -330,6 +395,26 @@ vermögensgrundfreibetrag:
     1948: 150
     reference: Artikel 1 G. v. 20.07.2006 BGBl. I S. 1706.
 
+schonvermögen_bürgergeld:
+  name:
+    de: Schonvermögen beim Bürgergeld
+    en: Protected wealth for the citizen's income
+  description:
+    de: Seit der Einführung des Bürgergelds zum 01.01.2023 gilt ein
+        Vermögensfreibetrag von 15.000 Euro pro Mitglied der Bedarfsgemeinschaft
+        nach Ablauf der Karenzzeit von einem Jahr. Während der Karenzzeit
+        ist ein Vermögen von 40.000 Euro für die erste Person einer Bedarfsgemeinschaft
+        und 15.000 Euro für jede weitere erlaubt.
+    en: Since the introduction of the citizen's allowance as of 01.01.2023,
+        every member of the need community is allowed to own 15,000 Euro after
+        the end of the waiting period of 1 year. During the waiting period
+        wealth of 40,000 euros are allowed for the first person in a needs-based
+        community and 15,000 euros for each additional person.
+  unit: Euro
+  2023-01-01:
+    vor_karenzzeit: 40000
```

Comment:
Maybe rather `während_karenzzeit`?

Reply:
Do I understand correctly that the other vermögensfreibetrag-variables are no longer used from 2023 on? Then we can set them to `null` from this time on (would be great if you can check this for all the alg 2 parameters)